### PR TITLE
MOR-323: Fixed member listing (showing current of total) and showing community manager in list

### PR DESCRIPTION
**Pull request by kaspermarkus**
…

### CODE REVIEW

**Comment by javihernandez** [comment]
can you please update with current master?

**Comment by kaspermarkus** [comment]
done

File: src/components/dashboardV2/MembersList.vue [hunk 1]
```diff
@@ -14,6 +14,7 @@
           <b v-if="member.bar_id === activeBarId">{{ member.first_name }} {{ member.last_name }}</b>
```
Comment (javihernandez) [inline src/components/dashboardV2/MembersList.vue:14]: A couple of things:

1.- Read the comment at line 10.
If you want to render the CM as a member, you need to update the if clause to be ">0" or (my preferred) to get rid of the complete if statement at line 11 (and the related else block at line 32/33) so we don't check for the amount of people in the community.

2.- Beyond that, I see an error in the console while trying to render the CM in the list of members. And this is the information that the server returns about the CM: 
`
{"id":"910c177d-fc8a-4ded-99b0-c6cf45c0bf41","first_name":"Pat","last_name":Smith,"role":"manager","bar_id":null,"state":"active"}
`
As you can see, the CM doesn't have a bar assigned, and I assume that we need to add the default one to get rid of such errors. I'd say we need to update the CM's profile as soon as the default bar is created.

File: src/views/Dashboard.vue [hunk 2]
```diff
@@ -203,6 +203,8 @@ export default {
             .then((resp) => {
               this.list = this.autoHideDetails(bars, true)
               this.members = resp.data.members
+              // ensure all members have a bar_id associated, else set default
+              this.members.map(m => { return m.bar_id ? m : Object.assign(m, { bar_id: this.community.default_bar_id })});
```
Comment (javihernandez) [inline src/views/Dashboard.vue:207]: I see you're trying to do such thing (ensuring the members including the CM have a bar assigned), but I guess it's not working properly ? Or am I getting a different but related error?

File: src/components/dashboardV2/MembersList.vue (hunk 1 above)
Comment (kaspermarkus) [reply src/components/dashboardV2/MembersList.vue:14]: Good catch! I've fixed as per your recommendation on #1. For #2 I've done a duplicate fix of in the dashboard... It probably should have been resolved somewhere earlier and members passed around from a central point, but with the timecrunch I think this is a sensible solution (it's only a 1-liner that is duplicated)

Comment (kaspermarkus) [reply src/components/dashboardV2/MembersList.vue:14]: ok, did a further update... It looks like members are fetched in two places, fixed the bar_id thing where data is loaded in both places.. I dont think this solution is too crazy - and a more detailed fix would be too extensive with the given timeframe

File: src/views/Dashboard.vue (hunk 2 above)
Comment (kaspermarkus) [reply src/views/Dashboard.vue]: The error was unrealted to this (see above).

Things should be working properly now

**Comment by kaspermarkus** [comment]
ready for another round, @javihernandez

File: src/components/dashboardV2/MembersList.vue [hunk 3]
```diff
@@ -71,6 +71,7 @@ export default {
   },
   computed: {
     orderedMembers: function () {
+      console.log("Kasper");
```
Comment (javihernandez) [inline src/components/dashboardV2/MembersList.vue:74]: typo, sed s/Kasper/DAMNYOUKASPARNET :stuck_out_tongue_closed_eyes:

Comment (klown) [reply src/components/dashboardV2/MembersList.vue:74]: Also, it should be: `console.error("DAMNYOUKASPARNET");`.

Perhaps add a `console.log("Kasper needs a beer");`

**Pull request merged**